client/my-sites/pages/main.jsx

```diff
@@ -106,11 +104,27 @@ module.exports = React.createClass( {
 	},
 
 	_setWarning( selectedSite ) {
+		const { translate, displayWarningNotice } = this.props;
 		if ( selectedSite && selectedSite.jetpack && ! selectedSite.hasMinimumJetpackVersion ) {
-			notices.warning(
-				this.translate( 'Jetpack %(version)s is required to take full advantage of all page editing features.', { args: { version: config( 'jetpack_min_version' ) } } ),
-				{ button: this.translate( 'Update now' ), href: selectedSite.options.admin_url + 'plugins.php?plugin_status=upgrade' }
+			displayWarningNotice(
+				translate(
+					'Jetpack %(version)s is required to take full advantage of all page editing features.',
+					{ args: { version: config( 'jetpack_min_version' ) } }
+				),
+				{ button: translate( 'Update now' ), href: selectedSite.options.admin_url + 'plugins.php?plugin_status=upgrade' }
```

Comment:
Totally nitpicking (and you can feel free to ignore as it hasn't been introduced in this PR), but I really prefer keeping these on separate lines:

```
{
	button: translate( 'Update now' ),
	href: selectedSite.options.admin_url + 'plugins.php?plugin_status=upgrade'
}
```